flask_admin/translations/zh_Hans_CN/LC_MESSAGES/advanced.po

```diff
@@ -702,7 +702,7 @@ msgid ""
 "admin/flask-admin/tree/master/examples/peewee."
```

Comment:
Unfortunately the URL is split across lines here so would also need to update these - might need a bit of manual checking for any other instances 👀